call.go

```diff
@@ -147,15 +150,49 @@ func Invoke(ctx context.Context, method string, args, reply interface{}, cc *Cli
 	return invoke(ctx, method, args, reply, cc, opts...)
 }
 
+const defaultClientMaxReceiveMessageSize = 1024 * 1024 * 4
+const defaultClientMaxSendMessageSize = 1024 * 1024 * 4
```

Comment:
Move those to clientconn.go, around line 106? Since they are shared by Unary and Streaming.
Remove "client" from the variable name.
And do
```go
const (
    defaultMaxReceiveMessageSize = 1024 * 1024 * 4
    defaultMaxSendMessageSize = 1024 * 1024 * 4
)
```